Dockerfile.debug

```diff
@@ -0,0 +1,42 @@
+FROM golang:1.15.2 AS build
+WORKDIR /contour
+
+ENV GOPROXY=https://proxy.golang.org
+COPY go.mod go.sum /contour/
+RUN go mod download
+
+COPY cmd cmd
+COPY internal internal
+COPY apis apis
```

Comment:
need to add the `pkg` dir as well now - see recent commit https://github.com/projectcontour/contour/commit/6668c5913cf73a32293edac813419771df5aaf53#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557